presto-main/src/main/java/com/facebook/presto/sql/planner/RowExpressionVariableInliner.java

```diff
@@ -31,19 +31,19 @@
         extends RowExpressionRewriter<Void>
 {
     private final Set<String> excludedNames = new HashSet<>();
-    private final Function<VariableReferenceExpression, RowExpression> mapping;
+    private final Function<VariableReferenceExpression, ? extends RowExpression> mapping;
```

Comment:
Why these changes?

Reply:
We can have both `Map<VariableReferenceExpression, VariableReferenceExpression>` and `Map<VariableReferenceExpression, RowExpression>`.